### DISCUSSION TITLE
Unable to configure GraphQL Schema

### DISCUSSION THREAD

**Discussion by jipperinbham** [open-telemetry/opentelemetry-ruby#1135] [Q&A]
When trying to explicitly set the schema via the following:

```ruby
  OpenTelemetry::SDK.configure do |c|
    c.use_all({
      'OpenTelemetry::Instrumentation::PG' => {
        db_statement: :obfuscate
      },
      'OpenTelemetry::Instrumentation::ActiveModelSerializers' => {
        enabled: false
      },
      'OpenTelemetry::Instrumentation::GraphQL' => {
        schemas: [FlySchema]
      }
    })
  end
```

I get the following error when the server starts up or when running tests:

```
OpenTelemetry error: unexpected configuration error due to Unknown type :hash - OpenTelemetry::SDK::ConfigurationError
```

Here is an excerpt of the schema (located at `/app/graphql/fly_schema.rb`):

```ruby
class FlySchema < GraphQL::Schema

  use Timeout, max_seconds: 30

end
```

**Comment by arielvalentin** [comment]
@jipperinbham It is difficult to tell what could be wrong here based on the code snippet. 

Would you be amenable to opening a PR or example repo with a test case that reproduces this issue?

Also, have you considered adding a custom `error_handler` to extract additional information from the exception?

Here is the default one for reference: https://cs.github.com/open-telemetry/opentelemetry-ruby/blob/ad51972e9bdc8ead0e69642056a4935683464dfc/api/lib/opentelemetry.rb?q=error_handler#L36
